MdeModulePkg/Core/Pei/Dispatcher/Dispatcher.c

```diff
@@ -5,9 +5,325 @@ Copyright (c) 2006 - 2024, Intel Corporation. All rights reserved.<BR>
 (C) Copyright 2016 Hewlett Packard Enterprise Development LP<BR>
 SPDX-License-Identifier: BSD-2-Clause-Patent
 
+Copyright (c) Microsoft Corporation.
+
 **/
 
 #include "PeiMain.h"
+#include <Library/PerformanceLib.h>
+
+// MU_CHANGE [BEGIN] - Delayed Dispatch
+
+/**
+ * DelayedDispatchDispatcher
+ *
+ * Delayed Dispach cycle (ie one pass) through each entry, calling functions when their
+ * time has expired.  When UniqueId is specified, if there are any of the specified entries
+ * in the dispatch queue during dispatch, repeat the DelayedDispatch cycle.
+ *
+ * @param DelayedDispatchTable  Pointer to dispatch table
+ * @param OPTIONAL              UniqueId used to insure particular time is met.
+ *
+ * @return BOOLEAN
+ */
+BOOLEAN
+DelayedDispatchDispatcher (
+  IN DELAYED_DISPATCH_TABLE  *DelayedDispatchTable,
+  IN EFI_GUID                *UniqueId           OPTIONAL
+  );
+
+/**
+ * Delayed Dispatch Notify function for EndoPei
+ *
+ * @return EFI_STATUS EFIAPI

```

Comment:
Can you please clean up the return status? `EFIAPI` doesn't make sense.

Reply:
Updated to `@return EFI_STATUS`